Headers/Foundation/NSObjCRuntime.h

```diff
@@ -286,13 +286,20 @@ DEFINE_BLOCK_TYPE(NSComparator, NSComparisonResult, id, id);
 /**
  * Declare Apple availability macros for compatibility purposes as no-ops.
  */
-#ifndef NS_CLASS_AVAILABLE
-#define NS_CLASS_AVAILABLE(x, y)
-#endif
```

Comment:
I added that macro myself some time ago and the `#ifndef` was mostly due to initial paranoia; I don't think this is ever defined anywhere else though and they seemed to clutter the code.